Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
I am not very familiar with `Makefile`s. Is there any possibility of placing doc-related things inside a `docs` dir or something like that?

Having a `Makefile` in the root dir of the repo is a little bit strange. Someone from the outside seeing the Makefile in the root dir might think that the whole app is built using Make. If the answer is no, then it's no problem.

Reply:
And then somehow calling `make` from the `Pipfile`, but pointing to `docs/Makefile`

Reply:
(This is not a necessary change though, just a question)

Reply:
I'm thinking if having this Makefile is actually really necessary. Perhaps we can just keep the commands on pipenv (as far as we can).

Reply:
Yeah, if that approach is possible I would vote for that!

Reply:
We can get rid of the Makefile. I can do it :)